src/strands/tools/registry.py

```diff
@@ -24,244 +24,318 @@
 
 
 class ToolRegistry:
-    """Central registry for all tools available to the agent.
+    """Agent Tool CRUD Interface.
```

Comment:
I think the "Central registry for all tools available to the agent." hits closer to the mark TBH. As someone reading through the code, the next sentence tells me the same thing so I'd rather see something like:

```
Contains the collection of tools available to the agent.
```